beginner_source/fgsm_tutorial.py

```diff
@@ -1,93 +1,71 @@
 # -*- coding: utf-8 -*-
 """
-Adversarial Example Generation
+적대적 예제 생성(Adversarial Example Generation)
 ==============================
 
-**Author:** `Nathan Inkawhich <https://github.com/inkawhich>`__
+**저자:** `Nathan Inkawhich <https://github.com/inkawhich>`__
+**번역:** `BONGMO KIM <https://github.com/bongmo>`__
 
-If you are reading this, hopefully you can appreciate how effective some
-machine learning models are. Research is constantly pushing ML models to
-be faster, more accurate, and more efficient. However, an often
-overlooked aspect of designing and training models is security and
-robustness, especially in the face of an adversary who wishes to fool
-the model.
-
-This tutorial will raise your awareness to the security vulnerabilities
-of ML models, and will give insight into the hot topic of adversarial
-machine learning. You may be surprised to find that adding imperceptible
-perturbations to an image *can* cause drastically different model
-performance. Given that this is a tutorial, we will explore the topic
-via example on an image classifier. Specifically we will use one of the
-first and most popular attack methods, the Fast Gradient Sign Attack
-(FGSM), to fool an MNIST classifier.
+이 글을 읽고 있다면, 머신 러닝 모델이 얼마나 효과적인지 진가를 알 수 있을 것입니다.
+머신 러닝 연구는 ML(Machine Learming) 모델을 더욱 빠르고 정확하며 효율적이 되는 방향으로 진행 되고 있습니다.
+그러나, 모델을 설계하고 훈련하는 데 종종 간과되는 측면은 모델을 속이려하는 적을 대면했을 때의 보안과 견고성입니다.
```

Comment:
```suggestion
그러나 모델을 속이려하는 적에 대한 보안과 견고함은 모델을 설계하고 훈련할 때 종종 간과되는 부분입니다.
```

번역하기가 조금 까다롭네요... 한 번 문장 순서를 바꾸어보았습니다.